File: solutions/algorithms/easy/cs/1.TwoSum.cs
using System;

namespace ConsoleApp1
{
    public class Solution
    {
        public int[] TwoSum(int[] nums, int target)
        {
            int[] V = { };
            for (int i = 0; i < nums.Length; i++) 
            {
                for (int j = 0; j < nums.Length; j++)
                {
                    if (i != j)
                    {
                        if (nums[i] + nums[j] == target)
                        {
                            return new int[] { i, j };
                        }
                    }
                }
            }
            return V;
        }
    }

    internal class Program
    {
        static void Main(string[] args)
        {
            int[] nums = { 2, 7, 11, 15 };
            int target = 9;

            Solution s = new Solution();

            int[]sda = s.TwoSum(nums, target);

            foreach (var i in sda)
            {
                Console.WriteLine(i);
            }
        }
    }
}

File: solutions/algorithms/easy/python3/1.TwoSum.py
class Solution:
    def twoSum(self, nums: list, target: int) -> list:
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[j] == target - nums[i]:
                    return [i, j]

nums = [2,3,11,3,12,7]
target = 6

s = Solution()
print(s.twoSum(nums, target))   